tests/evaluator/haspi/test_haspi.py

```diff
@@ -90,15 +94,14 @@ def test_haspi_v2_better_ear_error(hl_left, hl_right, freqs) -> None:
     proc_left = np.random.uniform(-1, 1, int(sample_rate * 0.5))  # i.e. 500 ms of audio
     proc_right = np.random.uniform(-1, 1, int(sample_rate * 0.5))
 
-    with pytest.raises(ValueError):
-        haspi_v2_be(
-            reference_left=ref_left,
-            reference_right=ref_right,
-            processed_left=proc_left + ref_left,
-            processed_right=proc_right,
-            sample_rate=sample_rate,
-            audiogram_left=hl_left,
-            audiogram_right=hl_right,
-            audiogram_frequencies=freqs,
-            level=100,
-        )
+    score = haspi_v2_be(
+        reference_left=ref_left,
+        reference_right=ref_right,
+        processed_left=proc_left + ref_left,
+        processed_right=proc_right,
+        sample_rate=sample_rate,
+        audiogram_left=Audiogram(levels=hl_left, frequencies=freqs),
+        audiogram_right=Audiogram(levels=hl_right, frequencies=freqs),
+        level=100,
+    )
+    assert score == pytest.approx(expected_score)
```

Comment:
Here are mising the `rel=pytest.rel_tolerance, abs=pytest.abs_tolerance` params in pytest.approx